### ISSUE TITLE
Upgrade from 3.11 to 3.12 causes wazuh-service can't start

### ISSUE THREAD

**Issue by jotacarma90** [wazuh/wazuh#4752]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 3.12 | All | Manager | All | All |

## Description
There is an error when trying to upgrade from version 3.11 to 3.12, in a manager. It installs and upgrades correctly, but when you try to start the wazuh-service, you get this error:
```
2020/03/18 08:30:13 ossec-analysisd: ERROR: Duplicate rule ID:65500
2020/03/18 08:30:13 ossec-analysisd: CRITICAL: (1220): Error loading the rules: 'ruleset/rules/0630-mcafee_epo_rules.xml'.
ossec-analysisd: Configuration error. Exiting
```

It must be an error when installing the directory:
`/var/ossec/ruleset/rules`.
I have checked it and it only happens in the managers, the update in an agent works correctly. At the moment the same thing happens in CentOS and Ubuntu.

**Comment by antoniomanuelfr** [comment]
The problem is caused because after the update some rules are duplicated in the directory `/var/ossec/ruleset/rules`. 
After watching the rules that differs from one installation to the other are: 

- `0625-mcafee_epo_rules.xml`
- `0675-panda-paps_rules.xml` 
- `0680-checkpoint-smart1_rules.xml` 

This rules have been renamed in the new version and after the update process the old rules are not removed.

**Comment by danimegar** [comment]
This issue was fixed in the commit 52ea74295f6cd467fe097e06efca4c4ec01f248f so it can be closed.
